README_Instructions.md

```diff
@@ -0,0 +1,32 @@
+# EdgeX-TAF execution in Pipeline
+
+This is created as an intro to understand how EdgeX-TAF will be executed in pipeline
+
+## trigger.sh
+
+This is main script which will be triggered from pipeline.
+
+user will need to run script as below from pipeline
+
+sh trigger.sh --service functionalTest/V2-API/<service_name> --auth <username/password> --port <port_number> --module <module> --ip <IP_address>
+
+   * service_name: this will be service name which is present under V2-API
+   * username/password: this will be username and password of board. if user will not provide then it will
+                        take default as admin/admin
+   * port: this is socket port. as of now we will take as 8002
```

Comment:
Right now you only specify 8002 (https).  Does it not work with 8001 (http)?  If that is the case, we should give a little more information on why.

Reply:
Mark it is working with 8001 port but as of now we are going with 8002 port.